cms/envs/common.py

```diff
@@ -83,7 +83,7 @@
 
     # If set to True, Studio won't restrict the set of advanced components
     # to just those pre-approved by edX
-    'ALLOW_ALL_ADVANCED_COMPONENTS': False,
+    'ALLOW_ALL_ADVANCED_COMPONENTS': True,
```

Comment:
Why enable 'ALLOW_ALL_ADVANCED_COMPONENTS'? If you need this for your tests then you can always patch django settings like "@patch.dict(settings.FEATURES, {'ALLOW_ALL_ADVANCED_COMPONENTS': True})"